### ISSUE TITLE
Edit footer to include Sitemap, Credits, Privacy Policy

### ISSUE THREAD

**Issue by daniellex0** [hackforla/website#1323]
### Dependency
~Updating the sitemap #1733~
~Privacy policy getting created https://github.com/hackforla/product-management/issues/26~

### Overview
We need to adjust the footer so that we can include links to the [sitemap](https://www.hackforla.org/sitemap/), [credits](https://www.hackforla.org/credits/), and privacy policy pages

This issue replaces #797 

### Action Items
- [x] Review [Figma file](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3417%3A3) with updated footer designs
- [x] Change layout of footer and add new [sitemap](https://www.hackforla.org/sitemap/), [credits](https://www.hackforla.org/credits/), and privacy policy (N/A) page links
- [ ] Do responsive design for mobile, tablet, desktop
   - [x] desktop
   - [x] mobile
   - [ ] tablet (nice to have but don't stop the update for this design)
- [ ] Comment out Privacy Policy for now (draft not finished yet)
- [ ] Please don't merge before design reviews and approves

### Resources/Instructions
- [Figma file](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3417%3A3) with updated footer designs
- [Sitemap page](https://www.hackforla.org/sitemap/)
- [Credits page](https://www.hackforla.org/credits/)

<img width="883" alt="Screen Shot 2021-03-24 at 2 09 00 AM" src="https://user-images.githubusercontent.com/67400366/112284023-fd958a00-8c45-11eb-8fed-c1a761381637.png">

<img width="226" alt="Screen Shot 2021-03-24 at 2 09 26 AM" src="https://user-images.githubusercontent.com/67400366/112284031-01291100-8c46-11eb-962f-2c4b543bce10.png">

**Comment by ExperimentsInHonesty** [comment]
What is the next step for this issue.  Isn't it to design the table view?

**Comment by daniellex0** [comment]
This has already been developed, but the Privacy Policy is still missing, and the sitemap needs your final approval before adding the new image and adding Sitemap page to the footer. Then this will be complete.

The new footer might need to be a separate issue for design, or can be added to this one- up to y'all.

Tagging @kristine-eudey to keep looped in

**Comment by ExperimentsInHonesty** [comment]
- This issue is complete enough.  The design has implemented, and the privacy policy is being added on another issue.  The footer is about to change substantially on this issue https://github.com/hackforla/website/issues/2554
